advanced_functionality/causal-inference/container/local_test/test_dir/input/data/training/heart_failure_clinical_records_dataset.csv

```diff
@@ -0,0 +1,300 @@
+age,anaemia,creatinine_phosphokinase,diabetes,ejection_fraction,high_blood_pressure,platelets,serum_creatinine,serum_sodium,sex,smoking,time,DEATH_EVENT
```

Comment:
@jkroll-aws Is it alright to download entire datasets into the repo or should the notebook download it from S3?

Reply:
Yes this needed to be changed - confirmed that CSVs have been removed from PR and now is downloading from S3.